internal/kafka/internal/services/clusters.go

```diff
@@ -30,6 +32,7 @@ type ClusterService interface {
 	Create(cluster *api.Cluster) (*api.Cluster, *apiErrors.ServiceError)
 	GetClusterDNS(clusterID string) (string, *apiErrors.ServiceError)
 	GetExternalID(clusterID string) (string, *apiErrors.ServiceError)
+	List(ctx context.Context) ([]*api.Cluster, *apiErrors.ServiceError)
```

Comment:
Let's add a comment on the behavior of this method something along listing the clusters using the org id information found in the context..

Reply:
will do. thanks